tests/testcastlerandom.pas

```diff
@@ -0,0 +1,75 @@
+{
+  Copyright 2012-2017 Michalis Kamburelis.
+
+  This file is part of "Castle Game Engine".
+
+  "Castle Game Engine" is free software; see the file COPYING.txt,
+  included in this distribution, for details about the copyright.
+
+  "Castle Game Engine" is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+  ----------------------------------------------------------------------------
+}
+
+{ Test CastleRandom unit. }
+unit testcastlerandom;
+
```

Comment:
Declare using uppercase, `TestCastleRandom`. The Lazarus will take care to save it using lowercase automatically, don't worry about it -- it will be saved as `testcastlerandom.pas`.